pkg/http/core/kubernetes/restart_test.go

```diff
@@ -73,6 +73,17 @@ var _ = Describe("Restart", func() {
 		})
 	})
 
+	When("the kind is not supported to restarted", func() {
+		BeforeEach(func() {
+			actionConfig.Operation.RollingRestartManifest.ManifestName = "not-supported-kind test-name"
```

Comment:
Shuldnt this be?:
actionConfig.Operation.RollingRestartManifest.ManifestName = “not-supported-kind”?

to match the message in Line #83

Reply:
This is one aspect of the Spinnaker API that I've always found weird. When referencing a manifest by name it is almost always referenced as `<kind> <name>`, which is what I'm doing here.